### PR TITLE
Add GraphQL XHR detection

### PR DESCRIPTION
**Pull request by ysm-dev**
https://graphql.org/

ex)
- https://www.loom.com/pricing
- https://www.apollographql.com/

### CODE REVIEW

File: src/technologies/g.json [hunk 1]
```diff
@@ -2069,6 +2069,7 @@
       "store-config": "graphqlMethod"
     },
     "oss": true,
+    "xhr": "/graphql$",
```
Comment (nurbek91) [inline src/technologies/g.json:2072]: I guess this won't work. 
description
Hostnames of XHR requests.

Comment (ysm-dev) [reply src/technologies/g.json:2072]: @nurbek91 oh I see.
Then how can I detect GraphQL on this site? : https://www.loom.com/pricing
This site looks using GraphQL.

Comment (nurbek91) [reply src/technologies/g.json:2072]: @ysm-dev  at this stage it is not possible
I don’t know why it was done this way, apparently there are some reasons

Comment (ysm-dev) [reply src/technologies/g.json:2072]: Thank you! @nurbek91

**Pull request closed without merging**